backend/app/api/v1/endpoints/overtime_records.py

```
@@ -0,0 +1,320 @@
+
+@router.post(
+    "/",
+    response_model=OvertimeRecordOut,
+    status_code=status.HTTP_201_CREATED,
+    summary="Create an overtime record",
+    description="Create a new overtime record with validation and notifications."
+)
+async def create_overtime_record_endpoint(
+    overtime: OvertimeRecordCreate,
+    request: Request,
+    current_user: Users = Depends(get_current_user),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)